master/buildbot/status/buildstep.py

```diff
@@ -396,13 +408,31 @@ def asDict(self, request=None):
             self.build.builder.status.getURLForThing(l) + args]
                 for l in self.getLogs()]
 
-        result["urls"] = self.getURLs()
-
+        result["urls"] = self.temporaryUrlConversion()
```

Comment:
@BarryUnity I think we need to add here the artifacts and dependencies as a new elements in the result with the new format